index.html

```diff
@@ -6,7 +6,37 @@
 <p>18F Guides is the repository for best practices across our teams. 
 Our guides are open source, and you're free to use them as you wish.
 Our hope is that other digital service teams — both inside and outside the government — 
-will adopt or modify the practices outlined here. </p> 
+will adopt or modify the practices outlined here.</p>
+
+<p>Here's a curated list of guides found both here and elswhere that will help you get started:</p>
+
+##Management
```

Comment:
Definitely think we should have some name standardization, but that probably requires another conversation. 

Not sure if the open source guide belongs in Data. Maybe Dev? Also, if we have a Design grouping, then we should take Design out of "Dev and Design."